src/web/components/StickyBottomBanner/PuzzlesBanner.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useEffect, useState } from 'react';
+import * as emotion from 'emotion';
+import * as emotionCore from '@emotion/core';
+import * as emotionTheming from 'emotion-theming';
+import { useHasBeenSeen } from '@root/src/web/lib/useHasBeenSeen';
+import {
+	getWeeklyArticleHistory,
+	logView,
+} from '@root/node_modules/@guardian/automat-client';
+import {
+	shouldHideSupportMessaging,
+	getArticleCountConsent,
+} from '@root/src/web/lib/contributions';
+import {
+	sendOphanComponentEvent,
+	submitComponentEvent,
+} from '@root/src/web/browser/ophan/ophan';
+import { getCookie } from '@root/src/web/browser/cookie';
+import { getZIndex } from '@root/src/web/lib/getZIndex';
+import { trackNonClickInteraction } from '@root/src/web/browser/ga/ga';
+import { WeeklyArticleHistory } from '@root/node_modules/@guardian/automat-client/dist/types';
```

Comment:
Is this right? This might be an automat thing but it sure looks odd so I thought I'd check!